### ISSUE TITLE
Won't launch

### ISSUE THREAD

**Issue by Kenzie09** [ScottMichaud/YandereSimulatorLauncher2#8]
My game Won't launch, it says i have everything downloaded and my PC Is able to Play it but it isn't launching, says i'm missing something,

**Comment by ScottMichaud** [comment]
Sorry for the late reply. I thought I already answered this one, but maybe I got distracted and closed the browser.

The problem description is a little vague, but it sounds like you might have a 32-bit version of Windows. Yandere Simulator dropped support for 32-bit Windows back in April.

To verify this:
- Hold the left or right Windows key and press R
- A little "Run" window should pop up... probably near the start menu but it could be anywhere.
- Type dxdiag and click OK
- If you see 32-bit then Yandere Simulator will not run on your machine.

If you see 64-bit, like in the screenshot that's highlighted in blue, then you are not affected by the problem that I am thinking about, and I will need more information to find out what's actually wrong. What does the specific error message say?

![image](https://user-images.githubusercontent.com/2958972/92200547-33a92300-ee48-11ea-8668-5f08b5be85c6.png)
